### PR TITLE
feat: Account class

### PR DESCRIPTION
**Pull request by RogerMo01**
What should I implement in al method called ToString() for this class??

### CODE REVIEW

File: OOP-Exercises/Account.cs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        {
+            return Balance;
+        }
+
+        public void Credit(int amount)
+        {
+            Balance += amount;
+        }
+
+        public void Debit(int amount)
```
Comment (LongJohnSilver1504) [inline OOP-Exercises/Account.cs:47]: use ternary expression here

File: OOP-Exercises/Account.cs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        {
+            if (amount > Balance)
```
Comment (LongJohnSilver1504) [inline OOP-Exercises/Account.cs:49]: use ternay expression here

File: OOP-Exercises/Account.cs (hunk 1 above)
Comment (RogerMo01) [reply OOP-Exercises/Account.cs:47]: Cleared Up!!!!

Comment (RogerMo01) [reply OOP-Exercises/Account.cs:47]: Repeated!

File: OOP-Exercises/Account.cs (hunk 2 above)
Comment (RogerMo01) [reply OOP-Exercises/Account.cs:49]: Repeated!

**Pull request merged**